whitepaper/Cosmology/wl.tex

```diff
@@ -10,152 +10,425 @@
+useful information in the wavefront sensors and the guide CCDs that may be used
+to regularize the PSF reconstruction in a visit.  We might read out guider CCDs
+in different ways to better monitor the atmosphere.
+
+
+\subsection{Deep vs Wide}
+
+As outlined above, many revisits to each field spanning many RotSkyPos angles
+aids the suppression of shear systematics.  For a given per-visit exposure time
+this leads to a deep survey.  There are several advantages to a deep survey over
+a shallow-wide survey for weak lensing science.  A strategy question for LSST is
+whether to go wide first and then deep, or the reverse.
+
+There are actually several drivers for depth over area, given fixed observing
+time and camera+telescope etendue.  A deep observing strategy maximizes the
+cosmological signal-to-noise ratio either by maximizing the signal or minimizing
+the noise.
+
+First, it boosts the amplitude of the cosmic shear signals due to the increased
+amplitude of the lensing kernel.  Given the same lens mass, the amplitude of
+lensing signals is a simple function of the two distances: observer-to-lens
+versus lens-to-source. For example, when a lens is at z = 0.5, the shear for a
+source at z = 1.1 is nearly twice that at z = 0.7, leading to a factor of 4
+ratio in correlation function amplitudes.  Thus, a deep survey can take
+advantage of the geometric effect of gravitational lensing more efficiently.
+
+Second, it enables a longer redshift baseline to break parameter degeneracies.
+For example, shallower surveys with wider area are not efficient in shrinking
+the length of the ``banana'' in the $\Omega_M$-$\sigma_8$ plane because of the
+$\sigma_8 \Omega_M^\alpha = \mathrm{const}$ degeneracy.  The deep strategy
+compensates for the loss of volume due to a reduced area by extending the volume
+along the line of sight.
+
+Third, deep surveys provide more useful galaxies per area.  This merit does not
+entirely overlap with the second point.  In addition to new sources at higher
+redshift, the fainter limiting magnitude enables detection of fainter galaxies
+at a given redshift and better signal-to-noise ratio.  Needless to say, the
+increased source density reduces the impact of shape noise caused by intrinsic
+ellipticity dispersion.
+
+Fourth, it mitigates the effects of intrinsic alignments (IA), an important
+theoretical systematic in precision cosmic shear.  Current studies
+\citep{Heymans2013} indicate that IA effects are dominated by luminous red
+galaxies (LRGs).  Because a deep survey can access fainter galaxies, the net IA
+systematic decreases because the fraction of the LRGs decreases at fainter
+limiting magnitude.  Using the approach of \citet{Joachimi2011}, at z = 0.5 the
+amplitude of the IA power spectrum is reduced by a factor of two for an increase
+in the limiting r magnitude from 24 to 27 mag.
+
+For cosmic shear the volume at 5-60 arcmin scales and over a wide range of
```

Comment:
So we're not going to benefit from cosmic shear or galaxy-galaxy lensing on several degree scales?   This is quite powerful for constraining cosmology at lower redshifts.